libs/ui-toolkit/src/components/tabs/tabs.tsx

```diff
@@ -18,12 +20,10 @@ export const Tabs = ({
   onValueChange,
   ...props
 }: TabsProps) => {
-  const [activeTab, setActiveTab] = useState<string | undefined>(() => {
-    if (defaultValue) {
-      return defaultValue;
-    }
-    return children.find((v) => v)?.props.id;
-  });
+  const [settingsOpened, setSettingsOpened] = useState(false);
```

Comment:
I'm wondering if each pane store (the ones that use the `createDataGridSlice`) should have a `createSettingsSlice`, which includes a simple boolean field for if the settings are open and a setter function. This has some advantages such as being open the settings for a specific pane from anywhere. It also means we don't have to tie the tabs component to our 'pane' logic

Reply:
Settings are in popover now.